test/go/iterative_verifier_integration_test.go

```diff
@@ -1,293 +0,0 @@
-package test
-
-import (
-	"database/sql"
-	"testing"
-
-	"github.com/Shopify/ghostferry"
-	"github.com/Shopify/ghostferry/testhelpers"
-	"github.com/siddontang/go-mysql/schema"
-	"github.com/stretchr/testify/assert"
-)
-
-func TestHashesSql(t *testing.T) {
-	columns := []schema.TableColumn{schema.TableColumn{Name: "id"}, schema.TableColumn{Name: "data"}, schema.TableColumn{Name: "float_col", Type: schema.TYPE_FLOAT}}
-	pks := []uint64{1, 5, 42}
-
-	sql, args, err := ghostferry.GetMd5HashesSql("gftest", "test_table", "id", columns, pks)
-
-	assert.Nil(t, err)
-	assert.Equal(t, "SELECT `id`, MD5(CONCAT(MD5(COALESCE(`id`, 'NULL')),MD5(COALESCE(`data`, 'NULL')),MD5(COALESCE((if (`float_col` = '-0', 0, `float_col`)), 'NULL')))) "+
-		"AS row_fingerprint FROM `gftest`.`test_table` WHERE `id` IN (?,?,?) ORDER BY `id`", sql)
-	for idx, arg := range args {
-		assert.Equal(t, pks[idx], arg.(uint64))
-	}
-}
-
-func TestVerificationFailsDeletedRow(t *testing.T) {
-	ferry := testhelpers.NewTestFerry()
-	iterativeVerifier := &ghostferry.IterativeVerifier{}
-	ran := false
-
-	testcase := &testhelpers.IntegrationTestCase{
-		T:           t,
-		SetupAction: setupSingleTableDatabase,
-		AfterRowCopyIsComplete: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			setupIterativeVerifierFromFerry(iterativeVerifier, ferry.Ferry)
-
-			err := iterativeVerifier.Initialize()
-			testhelpers.PanicIfError(err)
-
-			err = iterativeVerifier.VerifyBeforeCutover()
-			testhelpers.PanicIfError(err)
-		},
-		BeforeStoppingBinlogStreaming: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			ensureTestRowsAreReverified(ferry)
-		},
-		AfterStoppedBinlogStreaming: func(ferry *testhelpers.TestFerry, sourceDB, targetDB *sql.DB) {
-			deleteTestRowsToTriggerFailure(ferry)
-			result, err := iterativeVerifier.VerifyDuringCutover()
-			assert.Nil(t, err)
-			assert.False(t, result.DataCorrect)
-			assert.Regexp(t, "verification failed.*gftest.table1.*pks: (43)|(42)|(43,42)|(42,43)", result.Message)
-			ran = true
-		},
-		DataWriter: &testhelpers.MixedActionDataWriter{
-			ProbabilityOfInsert: 1.0 / 3.0,
-			ProbabilityOfUpdate: 1.0 / 3.0,
-			ProbabilityOfDelete: 1.0 / 3.0,
-			NumberOfWriters:     4,
-			Tables:              []string{"gftest.table1"},
-		},
-		Ferry: ferry,
-		DisableChecksumVerifier: true,
-	}
-
-	testcase.Run()
-	assert.True(t, ran)
-}
-
-func TestVerificationFailsUpdatedRow(t *testing.T) {
```

Comment:
This is basically the same as `TestVerificationFailsDeletedRow`, which is covered by the same case.